src/points/saga.ts

```diff
@@ -87,6 +90,7 @@ export function* getHistory({ payload: params }: ReturnType<typeof getHistorySta
         appendHistory: params.getNextPage,
         newPointsHistory: history.data.filter((record) => isClaimActivityId(record.activityId)),
         nextPageUrl: history.hasNextPage ? history.nextPageUrl : null,
+        pointsBalance: history.balance,
```

Comment:
i realized that we may have a small discrepancy when making the history endpoint returning the balance: when fetching the next history page we may update the balance, but the history feed will remain unchanged until the user pulls to refresh. so we will lose the matching between balance and history.

perhaps we should update the balance only when we are fetching the first page?

Reply:
what do you mean by "so we will lose the matching between balance and history"? like the sum of the history entries that are rendered on the bottom sheet will not add to the number shown on the home screen? i agree that might be a little strange, especially if the user e.g., redeems all their points (so the home screen shows 0 but the history screen shows positive points) or something like that.

Reply:
@jophish i mean that if the balance will suddenly update, the top of the history will not:
* from the start, we get the balance and the first page (we see the last balance and the last events)
* but then the user opens the bottom sheet and scrolls down
* we get the _updated balance_ and the next page (but not the page with those events that updated that balance)
* the user notices that the balance is updated
* scrolls to the top of the bottom sheet to see the new entries, but there are no new entries because they have to pull to refresh

this is why I'm proposing to update the balance only in sync with fetching the first page, so the user will see their 0 balance updated only when they pull to refresh